torch_np/_funcs_impl.py

```diff
@@ -913,6 +913,28 @@ def take_along_axis(arr: ArrayLike, indices: ArrayLike, axis):
     return torch.take_along_dim(arr, indices, axis)
 
 
+def put(a: ArrayLike, ind: Sequence[ArrayLike], v: ArrayLike, mode="raise"):
+    if mode != "raise":
+        raise NotImplementedError(f"{mode=}")
+
+    index = torch.concat(ind)
+    index[index < 0] += a.numel()  # normalise negative indices
```

Comment:
PyTorch supports negative indices, so this should not be necessary.

Reply:
(This was for making unique work on equivalent negative indices, but that's gone now so resolving this)